app/assets/javascripts/discourse/app/models/user.js

```diff
@@ -399,12 +399,14 @@ const User = RestModel.extend({
 
     ["muted", "regular", "watched", "tracked", "watched_first_post"].forEach(
       (s) => {
```

Comment:
`s` is so undescriptive here... `trackingProp` or something would be better, especially since it's used multiple times. I don't even know what `s` could stand for (I know this is not you, but may as well improve at same time)